braindecode/models/eegconformer.py

```diff
@@ -389,9 +422,10 @@ def __init__(self, final_fc_length, n_classes,
             self.classification = nn.Identity()
```

Comment:
here, assign `softmax`/`identity` to just `classification` variable and not `self.classification`, then when looking at the model layers `classification` will apear only once not twice